server/sonar-web/src/main/js/apps/users/form-view.js

```diff
@@ -55,7 +55,7 @@ export default ModalForm.extend({
     const scmAccounts = this.$('[name="scmAccounts"]').map(function () {
       return $(this).val();
     }).toArray();
-    return scmAccounts.filter(function (value) {
+    return scmAccounts.filter(value => {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove curly braces and "return" from this arrow function body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)